backend/src/tasks/test/dnstwist.test.ts

```diff
@@ -28,19 +28,25 @@ describe('dnstwist', () => {
                 {
                     "fuzzer": "Homoglyph",
                     "domain-name": "test-domain.one",
-                    "dns-a": ["21.22.23.24"]
+                    "dns-a": ["21.22.23.24"],
+                    "whois-created": "2020-07-23",
```

Comment:
```suggestion
                    "date-discovered"
```

Reply:
- If domain already has date-discovered, keep it
- Otherwise, set date-discovered to current date
- If domain no longer exists, remove it

Reply:
@aloftus23 can you also rename this PR / issue to reflect that we're no longer going to use whois?